### PR TITLE
Init Travis CI file.

### PR DESCRIPTION
**Pull request by frewsxcv**

### CODE REVIEW

**Comment by frewsxcv** [comment]
@steveklabnik i think travis ci needs to be turned on in their web ui for this to work

**Comment by steveklabnik** [comment]
It says "you do not have sufficient rights to enable Travis" wtf

**Comment by mgattozzi** [comment]
@steveklabnik lol what? How can you not have sufficient rights for your own repo? Hmmmmm you did sync and update your github repos with Travis yeah?

**Comment by steveklabnik** [comment]
I did.

I've emailed support, let's see.

**Comment by mgattozzi** [comment]
I wonder if it's just a weird issue with the repo having been private and now it not being private.

**Comment by steveklabnik** [comment]
It just had some latency I guess, it worked now. Building!

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+language: rust
```
Comment (steveklabnik) [inline .travis.yml]: you'll need a `rust: nightly` here too, as doxidize only runs on nightly

also an `os: - linux - osx` might be nice so we get both of them

**Comment by frewsxcv** [comment]
looks like we're hitting a compile error for handlebars-rust now:

```
error[E0518]: attribute should be applied to function
 --> /Users/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/handlebars-0.30.1/src/grammar.rs:3:10
  |
3 | #[derive(Parser)]
  |          ^^^^^^ not a function
error: aborting due to previous error
```

**Comment by mgattozzi** [comment]
Looks like there's a beta we could try out to make sure it compiles. Specific version is 1.0.0-beta.1

**Comment by frewsxcv** [comment]
alright y'all, this is passing now 🎉

- i had to pin to a nightly from a week ago since we are hitting https://github.com/rust-lang/rust/issues/49635 with the latest nightly
- i commented out a test that's failing on master https://github.com/steveklabnik/doxidize/issues/88

**Comment by steveklabnik** [comment]
Gah, thank you!

**Pull request merged**